spec/jobs/notify_slack_spec.rb

```diff
@@ -17,6 +19,29 @@
       response = Jobs::NotifySlack.new.execute({post_id: post[:id]})
       expect(response[0]).to eq("success")
     end
+
+    it "only if topic's category have a filter" do
+      ::PluginStoreRow.find_by(plugin_name: PLUGIN_NAME, key: "_category_*_#general").destroy
```

Comment:
Yes it is default filter. It will send notification for all categories. So to check single category's workflow I removed it.